observations/config.go

```diff
@@ -90,28 +98,25 @@ func (cfg *MetricsConfig) LoadJSON(raw []byte) error {
 }
 
 func (cfg *MetricsConfig) loadMetricsOptions(jcfg *jsonMetricsConfig) error {
-	var err error
-	cfg.EnableStats, err = strconv.ParseBool(jcfg.EnableStats)
-	if err != nil {
-		return err
-	}
-	cfg.PrometheusEndpoint = jcfg.PrometheusEndpoint
+	cfg.EnableStats = jcfg.EnableStats
+	endpointAddr, _ := ma.NewMultiaddr(jcfg.PrometheusEndpoint)
+	cfg.PrometheusEndpoint = endpointAddr
 
 	return config.ParseDurations(
 		metricsConfigKey,
 		&config.DurationOpt{
 			Duration: jcfg.StatsReportingInterval,
 			Dst:      &cfg.StatsReportingInterval,
-			Name:     "metrics_reporting_interval",
+			Name:     "reporting_interval",
```

Comment:
I think we usually put the name as `metric.reporting_interval` or something that helps you find the section in the config. but no big deal.